BenchBnb/BenchBnb/Views/Register/Register.cshtml

```diff
@@ -0,0 +1,31 @@
+﻿@model BenchBnb.FormModels.CreateRegister
+
+@{
+    ViewBag.Title = "Register";
+}
+
+<h2>Register</h2>
+
+
+@Html.ValidationSummary()
```

Comment:
I'd move this call to the `ValidationSummary` method inside of your HTML form.

Reply:
Fixed